### ISSUE TITLE
package copies delete the package metadata from the UI

### ISSUE THREAD

**Issue by beckermr** [Anaconda-Platform/anaconda-client#556]
This issue is more for the backend server than the client, but here it is. 

When I copy a package from one user to another, the doc url, dev url etc get deleted from the destination user's package info. Further, there is not patch operation for the destination package to restore this information. 

cc @jakirkham @kkraus14 @leofang

**Comment by csoja** [comment]
anaconda.org API should now support:
POST -  copy package to the destination user and If a conflict is found, 409 CONFLICT error will be raised with a message pointing to other copy methods.
PUT -  copy package to the destination user and If a conflict is found, existing metadata will be overwritten.
PATCH - copy package to the destination user and If a conflict is found, existing metadata will be extended with missing fields.

anaconda-client also has support of POST/PUT/PATCH copying in master - but this is not yet available in a released version.  We're hoping to get a new released version of anaconda-client out that includes these changes in the near future. (RE: https://github.com/Anaconda-Platform/anaconda-client/pull/559)

Please let us know if you run into any trouble with or questions about the new functionality.

**Comment by beckermr** [comment]
Thanks @csoja! I was wondering about PUT and PATCH for this endpoint `GET /package/{owner_login}/{package_name}`

**Comment by vshevchenko-anaconda** [comment]
@beckermr PUT/PATCH aren't supported for `/package/{owner_login}/{package_name}`. Support of PUT and PATCH for the copy endpoint described by @csoja was added to fulfill the need in restoring info after broken copying

**Comment by beckermr** [comment]
Yup. It'd be nice to have PUT and PATCH for the `/package/{owner_login}/{package_name}` too. This way we can fix borken metadata without having to upload a package.
